EIPS/eip-1.md

```diff
@@ -217,6 +217,132 @@ The current EIP editors are
+- a widely-used Ethereum client (specifications and/or implementation)
+- a widely-used security tool: formal verification, static analysis, EVM reverse engineering and (dis)assemblers, etc.
+
+If such a contribution stops, the editor cannot be an `assigned` editor for this category anymore.
+
+#### Standard Track EIP - Networking
+
+An editor must provide knowledge of peer-to-peer networking, proved by having a solid and on-going contribution to at least one widely-used peer-to-peer network library, in any programming language. Knowledge of p2p libraries that are currently used in Ethereum (whisper, Swarm) is preferred, but not mandatory.
+
+#### Standard Track EIP - Interface
+
+An editor must provide knowledge of the interfaces used in Ethereum. At least one of the following knowledge requirements must be true:
+
+1. Client API/RPC specifications and standards that are currently used in Ethereum. Knowledge of additional RPC standards is preferred. This is proven by having a solid and ongoing contribution to at least one of:
+- maintaining specifications for an RPC protocol
+- implementing Ethereum RPC protocols in a widely-used tool (e.g. Ethereum clients)
+
+2. ABI specification and implementation. This is proven by having a solid and ongoing contribution to ABI-related parts of widely-used Ethereum tools - e.g. [Solidity](https://github.com/ethereum/solidity) compiler.
+
+3. EVM-C interface. This is proven by having a solid and ongoing contribution to [EVMC](https://github.com/ethereum/evmc).
+
+#### Standard Track EIP - ERC
+
+An editor must provide proof of knowledge of already finalized ERC standards by:
+- having participated in discussions or authoring of `Final` ERC standards (more than three).
+- contributing to existing implementations of `Final` ERC standards (more than three).
+
+All of the above are required.
+
+#### Meta EIP
+
+An editor must provide proof of knowledge of the current Ethereum processes by:
+- having participated in discussions or authoring `Final` Meta EIPs
+- contributing to the implementation of such processes or supervising the implementation.
+
+#### Informational EIP
+
+An editor must provide proof of knowledge by:
+- having participated in discussions or authoring `Final` Informational EIPs
+
+As these EIPs are optional, any editor who fits the requirements from the Standard Track and Meta EIP categories can apply for this category.
+
+#### EIP Syntax Editor
+
+An editor must demonstrate knowledge of natural language syntax and semantics (currently, for English), by either:
+- having university or higher credentials in the study of the current EIP language
+- having editorial experience or being a technical writer for technical journals or well-known publications
+
+The `Syntax` editor cannot be an `assigned` editor for an EIP. However, he/she will review EIP proposals or changes to existing EIPs, when requested.
+
+#### Editor Review
```

Comment:
Is this the category for someone who approves/manages editors? Maybe change to "Editor Manager" or something similar

Reply:
"Editor Manager" is more clear. Done in https://github.com/ethereum/EIPs/pull/2172/commits/2e7358a5b9bad8e698382a22e204f130da42e0eb